### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by shaneutt**
Update Prometheus codeowners.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -5,3 +5,6 @@
 # Order is important: later rules override preceding rules
 
 * @mesosphere/sig-ksphere-catalog
+
+# Prometheus
+addons/prometheus/* @gracedo
```
Comment (joejulian) [inline CODEOWNERS]: These are not additive, the more specific rule replaces the global rule. So we'd prefer this:
```suggestion
addons/prometheus/* @mesosphere/sig-ksphere-catalog @gracedo
```

File: CODEOWNERS [hunk 2]
```diff
@@ -5,3 +5,6 @@
 # Order is important: later rules override preceding rules
 
 * @mesosphere/sig-ksphere-catalog
+
+# Prometheus
+addons/prometheus/* @mesosphere/sig-ksphere-catalog @gracedo @branden
```
Comment (makkes) [inline CODEOWNERS]:
```suggestion
addons/prometheus/* @mesosphere/sig-ksphere-observability 
```

Comment (makkes) [reply CODEOWNERS]:
```suggestion
addons/prometheus/* @mesosphere/sig-ksphere-catalog @mesosphere/sig-ksphere-observability 
```

Comment (makkes) [reply CODEOWNERS]: I'm fine with either one of my suggestions.

**Pull request merged**